clients/go/pkg/zbc/client.go

```diff
@@ -41,6 +41,8 @@ const InsecureEnvVar = "ZEEBE_INSECURE_CONNECTION"
 const CaCertificatePath = "ZEEBE_CA_CERTIFICATE_PATH"
 const KeepAliveEnvVar = "ZEEBE_KEEP_ALIVE"
 const GatewayAddressEnvVar = "ZEEBE_ADDRESS"
+const GatewayHostEnvVar = "ZEEBE_HOST"
+const GatewayPortEnvVar = "ZEEBE_PORT"
```

Comment:
This is used in `zbctl` but it's ignored in the Go client. It should also be possible to use these when configuring the client

Reply:
Done